bodhi/server/services/comments.py

```diff
@@ -155,15 +155,15 @@ def query_comments(request):
     update_owner = data.get('update_owner')
     if update_owner is not None:
```

Comment:
What will happen if more conditions are true like `update_owner` and `ignore_user`? I'm not sure if this is even possible. If not, then fine, but otherwise, there should be test for this.

Reply:
In that case the result will display all comments in updates owned by `update_owner` list, made by users that are not `ignore_user` list. It will act in the same manner as now:
https://bodhi.stg.fedoraproject.org/comments/?ignore_user=bodhi&update_owner=mattia
The only thing that will change is that now `ignore_user` and `update_owner` can be made of multiple users.
And I don't see the reason for a test with multiple conditions, since each condition acts like the single one.

The odd thing is that (as now) you can pass both `user` and `ignore_user` conditions to the query, but since this is stupid and doesn't trigger any error (it will return an empty list) I don't think we need to explicitly forbid that:
https://bodhi.stg.fedoraproject.org/comments/?ignore_user=mattia&user=mattia